### ISSUE TITLE
Doesn't work with TiddlyWiki v5.1.22

### ISSUE THREAD

**Issue by weecks-kanaltechnik** [felixhayashi/TW5-TiddlyMap#344]
After installing and reloading it, it breaks my current TW5 5.1.22 like shown in the screenshot. 

May I hope you will update it to TW5.1.22 soon?

**Comment by danielo515** [comment]
Sorry for step in (I'm not the maintainer of this repo), but I have been updating some of my plugins recently for 5.1.22 and the amount of breaking changes made it a very unpleasant experience. In fact, I had to report a bunch of bugs.
What is exactly what you want from 5.1.22, I think it will be better to wait to the next version.

**Comment by weecks-kanaltechnik** [comment]
Hello Daniel, no problem at all. On the contrary, I'm glad to hear it! Thank you for stepping in. I will take a break from 5.1.22 till the next minor or major release? I wanted to use the TiddlyMap tool for my mind maps.

Von: Daniel Rodríguez Rivero <notifications@github.com>
Gesendet: Samstag, 25. April 2020 19:13
An: felixhayashi/TW5-TiddlyMap <TW5-TiddlyMap@noreply.github.com>
Cc: Weecks, Hans-Peter <HPWeecks@weecks-kanaltechnik.de>; Author <author@noreply.github.com>
Betreff: Re: [felixhayashi/TW5-TiddlyMap] Doesn't work with TiddlyWiki v5.1.22 (#344)


Sorry for step in (I'm not the maintainer of this repo), but I have been updating some of my plugins recently for 5.1.22 and the amount of breaking changes made it a very unpleasant experience. In fact, I had to report a bunch of bugs.
What is exactly what you want from 5.1.22, I think it will be better to wait to the next version.

**Comment by felixhayashi** [comment]
Hi @weecks-kanaltechnik I will try to reproduce the errors soon, I did not check the compatibility with 5.1.22 yet.

> I have been updating some of my plugins recently for 5.1.22 and the amount of breaking changes made it a very unpleasant experience.

Thanks @danielo515 for this information!

**Comment by felixhayashi** [comment]
Hi @weecks-kanaltechnik 

I just tested installing TiddlyMap (as described in the installation instructions at http://tiddlymap.org/) within TiddlyWiki 5.1.22 and everything worked fine for me.

I also upgraded my personal wikis that have TiddlyMap installed to 5.1.22 and didn't experience any problems.

I will close this issue therefore as I assume it is an issue with underlying plugins apart from TiddlyMap.
